src/Sarif.Viewer.VisualStudio/Sarif/SarifLog.extensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis.Sarif;
+
+namespace Microsoft.Sarif.Viewer.Sarif
+{
+    /// <summary>
+    /// Extension methods for the <see cref="SarifLog"/> class used by the viewer.
+    /// </summary>
+    internal static class SarifLogExtensions
+    {
+        static internal bool HasNoResults(this SarifLog sarifLog)
```

Comment:
So having functions that return negative flags is a bit confusing when they are combined with boolean operators.

I.e. 
if (!HasResults()) is a lot easier to understand mentally then if (!HasNoResults())

can we perhaps change this function to return a positive name rather than a negative one?

(I had to re-read line 22 like 10 times to make sure I understood it)

Reply:
+100. Fixed.

---
In reply to: [523438956](https://github.com/microsoft/sarif-visualstudio-extension/pull/272#discussion_r523438956) [](ancestors = 523438956)